src/libraries/Microsoft.Extensions.DependencyInjection.Abstractions/src/Microsoft.Extensions.DependencyInjection.Abstractions.csproj

```diff
@@ -1,18 +1,30 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>netstandard2.1;netstandard2.0;net461</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent);netstandard2.1;netstandard2.0;net461</TargetFrameworks>
     <EnableDefaultItems>true</EnableDefaultItems>
     <Nullable>enable</Nullable>
     <PackageDescription>Abstractions for dependency injection.
 
 Commonly Used Types:
 Microsoft.Extensions.DependencyInjection.IServiceCollection</PackageDescription>
+    <!-- Use targeting pack references instead of granular ones in the project file. -->
+    <DisableImplicitAssemblyReferences>false</DisableImplicitAssemblyReferences>
   </PropertyGroup>
 
   <ItemGroup>
     <Compile Include="$(CommonPath)Extensions\ParameterDefaultValue\ParameterDefaultValue.cs"
              Link="Common\src\Extensions\ParameterDefaultValue\ParameterDefaultValue.cs" />
+  </ItemGroup>
+
+  <ItemGroup Condition="$([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETCoreApp'">
```

Comment:
@safern @ViktorHofer - for my education, what's the difference between this and:

```xml
<ItemGroup Condition="'$(TargetFrameworkIdentifier)' == '.NETCoreApp'">
```

?

The latter seems easier to read/understand.

Reply:
On PropertyGroup/properties the `$([MSBuild]::GetTargetFrameworkIdentifier('$(TargetFramework)')) == '.NETCoreApp'` expression needs to be used as the `TargetFrameworkIdentifier` property isn't available at that point (it's set in the sdk's .targets file). For items, that expression doesn't need and IMO shouldn't be used as it's harder to read and takes longer (function exec vs property comparison).

Reply:
Ah, nice. Thanks for the explanation. That makes sense.

Reply:
So this line should be changed...